src/main/java/edu/hm/hafner/analysis/IssueBuilder.java

```diff
@@ -497,6 +497,12 @@ public IssueBuilder copy(final Issue copy) {
      * @return the created issue
      */
     public Issue build() {
+        if (lineRanges != null && lineRanges.size()>1) {
+            LineRange firstRange = lineRanges.get(0);
+            if (firstRange.getStart() == lineStart && firstRange.getEnd() == lineEnd) {
+                lineRanges.remove(0);
+            }
```

Comment:
And here you need to check the other direction as well. If the lineStart is not set, set it